helm/rbac-operator/templates/_helpers.tpl

```diff
@@ -19,10 +19,9 @@ Common labels
 {{- define "labels.common" -}}
 app: {{ include "name" . | quote }}
 {{ include "labels.selector" . }}
-app.giantswarm.io/branch: {{ .Values.project.branch | quote }}
```

Comment:
Architect provided these values. Abs does not do that anymore. 
However, they were not used anywhere, so I decided to remove these selectors.